tests/test_git.py

```diff
@@ -0,0 +1,13 @@
+"""
+Unit-tests for git
+"""
+
+import commodore.git as git
+
+
+def test_reconstruct_api_response(tmp_path):
+    repo_path = tmp_path / 'test-repo'
+    repo_path.mkdir()
+    repo = git.create_repository(repo_path)
+    output = git.stage_all(repo)
+    assert output != ""
```

Comment:
I'm not sure I understand what's happening here. Do you:

1. create directory "test-repo"
1. run `git init` inside of it
1. stage the (empty) change set

That needs to be empty. True. :thinking:

Reply:
It's just a very basic test and makes sure no exceptions are thrown.

Reply:
My point is, there's too much going on for a test. Just as some feedback.